client/gutenberg/extensions/related-posts/edit.jsx

```diff
@@ -40,6 +70,20 @@ class RelatedPostsEdit extends Component {
 		const displayPosts = postsToDisplay.slice( 0, postsToShow );
 		const previewClassName = 'related-posts__preview';
 
+		const displayMockupPosts = posts.length ? false : true;
```

Comment:
nitpick that you probably could have come come to on your own, please ignore if you want

```js
const displayMockupPosts = ! posts.length;
```

Reply:
Yeah. Ternary logic seems to be the norm in a lot of our code and this simply seemed more readable, but this makes more sense. Thanks!

Reply:
> Ternary logic seems to be the norm in a lot of our code

totally! and I don't want to imply you should avoid them - I _always_ prefer expressions over statements (ternary vs. conditional structures).

in this case though it's somewhat repetitive which is why I pointed it out; consider the following construct which appears with startling regularity around our various codebases…

```js
let flag;
if ( condition ) {
	flag = true;
} else {
	flag = false;
}
```

that's all I'm pointing out. in these cases `flag` is either `condition` (or for type safety) `!! condition` _or_ `! condition`

😄

Reply:
I just remember my Ruby linter shouting at me fir things like this :)

Reply:
> my Ruby linter shouting at me

oh no! why? I'm not a Rubyist - are there risks in using basic logical assignments?

in Javascript it's mainly a type-risk related to short-circuiting

```js
const a = 'Rosebud';
const b = 'snow';
const c = a || b; // c === 'Rosebud'
```

```php
$a = 'Rosebud';
$b = 'snow';
$c = $a || $b; // $c === true
```